### ISSUE TITLE
Animation Export Not Working

### ISSUE THREAD

**Issue by G0J1** [xavier150/Blender-For-UnrealEngine-Addons#171]
Hello,
Sometimes when I try to export animations, Blender just crashes. It was originally working but for some reason recently it does not work. I have no idea why, especially since there is the odd occasion where it does work. I don't know if it's an error on my end though.

**Comment by xavier150** [comment]
Hello, Thanks for the report! you can send me the blend file ? I will try to reproduce on my side.

**Comment by G0J1** [comment]
[2023Kaiju1-1.zip](https://github.com/xavier150/Blender-For-UnrealEngine-Addons/files/11220092/2023Kaiju1-1.zip)
 Here is the zip, thanks for the help and just this addon in general!

**Comment by xavier150** [comment]
Hello @G0J1 
Sorry for the delay. 
After many tests I found it, it an error from Blender. On your blender file when python script communicate with a copy of bpy.types.NlaStripFCurves it produce a crash.

I fixed it by using an Proxy class. I updated the dev branch. Try it that should be fixed now.
Dev Branch -> https://github.com/xavier150/Blender-For-UnrealEngine-Addons#download-and-installation-from-dev-branch

I will close this issue when I release update v0.4.2
